### ISSUE TITLE
License issue when using LVL and APK signature

### ISSUE THREAD

**Issue by creativetrendsapps** [javiersantos/PiracyChecker#19]
For some reason when using this library my app is reporting that the license isn't valid. I've implemented the play store check, the app is then downloaded from the play store and is returning that the app is invalid. Same with my signing key and if I use the license check. I'm not sure what I'm doing wrong.

**Comment by javiersantos** [comment]
Are you using the latest v1.0.1 update? The previous update had an issue with the license checker.

**Comment by creativetrendsapps** [comment]
lol, I should have said the previous version. I'll try this new version and let you know my results.

**Comment by creativetrendsapps** [comment]
Still the same issue. This works with my debug key perfectly, this doesn't, however, work with my production key which is strange.

**Comment by javiersantos** [comment]
Does your log show something? 

Please make sure that the signature retrieved by this method `PiracyCheckerUtils.getAPKSignature(this)` is the same as the signature that you are including in `.enableSigningCertificate("...")` when the signed APK with the production key is running. That is a common problem.

You can also use a buildConfigField value in the build.gradle to avoid modifying that value every time that the APK is compiled with the production key.

##### build.gradle
```gradle
android {
    ...
    buildTypes {
        release {
            ...
            buildConfigField 'String', 'APK_SIGNATURE', 'your_release_signature'
        }

        debug {
             ...
             buildConfigField 'String', 'APK_SIGNATURE', 'your_debug_signature'
        }
    }
}
```

```Java
new PiracyChecker(this)
    .enableSigningCertificate(BuildConfig.APK_SIGNATURE)
    ...
    .start();
```

**Comment by creativetrendsapps** [comment]
I'll try this and see if it is the proper solution.

**Comment by creativetrendsapps** [comment]
I'm thinking maybe it's the way APK's can be signed with Nougat? That's the last device I tried it on.

**Comment by javiersantos** [comment]
I'm not able to reproduce the issue on Nougat. Could you show me your configuration of the library? (remove specific values like your Base 64 license key or APK signature).

**Comment by creativetrendsapps** [comment]
@javiersantos, fixed! I had the wrong APK signature when checking. Fixed with: PiracyCheckerUtils.getAPKSignature(this). Thank you for this great library!

**Comment by javiersantos** [comment]
Thank you for letting me know!
